Core/ELF/ParamSFO.cpp

```diff
@@ -276,3 +280,34 @@ void ParamSFOData::ValueData::SetData(const u8* data, int size)
 	u_size = size;
 }
 
+std::string ParamSFOData::generateFakeID(std::string filename)
+{
+	// Generates fake gameID for homebrew based on it's folder name.
+	// Should probably not be a part of ParamSFO, but it'll be called in same places.
+	std::string file = PSP_CoreParameter().fileToStart;
+	if (filename != "")
+		file = filename;
+
+	std::size_t lslash = file.find_last_of("/");
+	file = file.substr(lslash + 1);
+
+	int letter, sumOfAllLetters = 0;
```

Comment:
Letter isn't used outside the loop, clearer to define it within the loop.

-[Unknown]